plugins/woocommerce/src/Internal/Admin/Settings/Utils.php

```diff
@@ -187,4 +187,62 @@ public static function order_map_change_min_order( array $order_map, int $new_mi
 
 		return $updated_map;
 	}
+
+	/**
+	 * Get the list of plugin slug suffixes used for handling non-standard testing slugs.
+	 *
+	 * @return string[] The list of plugin slug suffixes used for handling non-standard testing slugs.
+	 */
+	public static function get_testing_plugin_slug_suffixes(): array {
+		return array( '-dev', '-rc', '-test', '-beta' );
```

Comment:
We can include `-alpha` also, but what plugin was that? The one for beta testing WordPress? If that is the case, we don't need it because we are only dealing with plugins, not the WP core.

Reply:
Yes, it was for the WP core plugin. Sure, it's more like a nitpick.